src/main/views/users/tabs/deleteUserConfirm.njk

```diff
@@ -0,0 +1,22 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+<h2 class="govuk-heading-m">Remove roles for user: {{ userEmail }}</h2>
+
+<input type="hidden" id="user-id" name="user-id" value={{ userId }}>
```

Comment:
Exposing the user id in a hidden field might not be a good idea. I imagine we are able to see this if we go to the form and inspect the page? 

This ID could be useful information to a hacker or something - "putting on Alex Maynes voice"

Reply:
Think I would prefer another call to get user on idam, rather than storing the id on the page...need to think about this one a bit more 🤔